pom.xml

```diff
@@ -130,7 +132,11 @@
       <groupId>org.apache.maven.shared</groupId>
       <artifactId>maven-shared-utils</artifactId>
       <version>3.3.4</version>
-      <scope>runtime</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-archiver</artifactId>
+      <version>4.2.5</version>
```

Comment:
Sorry, but I have to, as am dropping m-shared-utils... so IMHO once this merged you should add your PR on top of it. W/o this change, due drop of m-shared-utils the build would fail :disappointed: